packages/ember-glimmer/tests/integration/helpers/log-test.js

```diff
@@ -0,0 +1,60 @@
+import { RenderingTest, moduleFor } from '../../utils/test-case';
+import { get } from 'ember-metal/property_get';
+import Logger from 'ember-metal/logger';
+
+var originalLog, logCalls;
+
+moduleFor('Helpers test: {{log}}', class extends RenderingTest {
+
+  constructor() {
+    super();
+
+    originalLog = Logger.log;
+    logCalls = [];
+    Logger.log = function(...args) {
+      logCalls.push(...args);
+    };
+  }
+
+  teardown() {
+    Logger.log = originalLog;
+  }
+
+  assertLog(assert, values) {
+    this.assertText('');
+    assert.strictEqual(logCalls.length, values.length);
+
+    for (let i = 0, len = values.length; i < len; i++) {
+      assert.strictEqual(logCalls[i], values[i]);
+    }
+  }
```

Comment:
(Not related to this pull request, and it should definitely block this: it bugged me a few times that we have to thread the `assert` bag to write a helper like this. I wonder if we can somehow get it in the constructor and set it as `this.assert` once at the beginning. Just some food for thought.)